picdar-export/src/main/scala/com.gu.mediaservice.picdarexport/lib/cleanup/UsageRightsOverride.scala

```diff
@@ -151,4 +151,8 @@ object UsageRightsOverride {
 
   def getUsageRights(copyrightGroup: String, metadata: ImageMetadata) =
     copyrightGroupToUsageRightsMap.get(copyrightGroup).flatMap(func => func(metadata))
+
+  def getOverrides(currentRights: UsageRights, picdarRights: UsageRights) = {
+
```

Comment:
Still need to figure what the logic is here.
Might be in order of importance:
* Media API overrides
* Picdar rights
* Media API auto rights

We will need to make sure we send over the overiddes from the Media API then.